grails-app/views/stockMovement/_create.gsp

```diff
@@ -0,0 +1,12 @@
+<html>
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+	<meta name="layout" content="custom" />
```

Comment:
Using the "custom" template might become problematic for you guys since it includes all of the old JS and CSS resources used on the existing pages. Use the main.gsp layout instead. Customize it to your needs and remove whatever doesn't need to be there. For example I think I saw a navbar item with the label "test" -- oops.  
```<meta name="layout" content="main" />```

Reply:
I used "custom" layout, because i wanted to have current menu integrated with upcomming features. At this point it is looking ok (we have current menu and our react component in body) . If you want, then i can use main layout there and add something similar to analytics navbar.

Reply:
It's ok for now. We can work on getting main.gsp in shape for the feature. My point was just that you'll start to see the YUI CSS and other CSS bleeding into the page as you start to add your react components. So your bootstrap components won't look exactly like you expect.

Reply:
I think it would be better to have a new task for this. Right now i could quickly swap layout to main and clean up it completly (it would have just body with our react component)

Reply:
Sounds good. Stay with custom and I'll create a new ticket for the main layout.